components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/impl/ExportApiUtil.java

```diff
@@ -60,19 +60,35 @@ public Response exportApiByParams(String name, String version, String providerNa
         //If not specified status is preserved by default
         boolean isStatusPreserved = preserveStatus == null || preserveStatus;
 
-        if (name == null || version == null || providerName == null) {
-            RestApiUtil.handleBadRequest("'name', 'version' or 'provider' should not be null", log);
+        if (name == null || version == null) {
+            RestApiUtil.handleBadRequest("'name' or 'version' should not be null", log);
         }
 
         try {
             //Default export format is YAML
             exportFormat = StringUtils.isNotEmpty(format) ? ExportFormat.valueOf(format.toUpperCase()) :
                     ExportFormat.YAML;
 
+            // Get currently logged in user's username and the domain
             userName = RestApiUtil.getLoggedInUsername();
+            apiRequesterDomain = RestApiUtil.getLoggedInUserTenantDomain();
+
+            // If provider name is not given
+            if (providerName.isEmpty()){
```

Comment:
Shall we use `StringUtils.isBlank` - [1] here?

[1] http://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isBlank(java.lang.String)

Reply:
+1 to use StringUtils.isBlank